azure-pipelines.yml

```diff
@@ -38,15 +38,21 @@ jobs:
     displayName: Build Windows Library
     pool:
       vmImage: windows-2022
+    variables: 
+      MSBuildSDKsPath: '/c/Users/VssAdministrator/AppData/Local/Microsoft/dotnet/6.0.200-preview.22055.15/sdks'
```

Comment:
not sure it will work. because you need preview version of VS to use preview version of .net. or you have to set 
![image](https://user-images.githubusercontent.com/33021114/150610359-d5503fe2-7280-4937-bba7-8214b40c2f2d.png)